spatialos-sdk/examples/project-example/main.rs

```diff
@@ -104,8 +110,25 @@ fn exercise_connection_code_paths(c: &mut WorkerConnection) {
             z: 0.0,
         },
     });
+    entity.add(improbable::EntityAcl {
```

Comment:
While you're here do you mind removing the unnecessary scoping on the `Position` and `Coordinates` added in lines 106 and 107?

Reply:
Also can remove the comment on line 92 😉